src/toolkits/style_transfer/style_transfer_data_iterator.cpp

```diff
@@ -0,0 +1,123 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#include "style_transfer_data_iterator.hpp"
+
+#include <core/data/image/io.hpp>
+#include <model_server/lib/image_util.hpp>
+
+namespace turi {
+namespace style_transfer {
+
+using neural_net::image_annotation;
+using neural_net::labeled_image;
+using neural_net::shared_float_array;
+
+flex_image get_image(const flexible_type& image_feature) {
```

Comment:
Helper functions local to a `.cpp` file should either be declared static or put inside an anonymous namespace

https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables